### DISCUSSION TITLE
Clerk manage account popup not clickable in mobile sidebar

### DISCUSSION THREAD

**Discussion by stargeneration-winston** [shadcn-ui/ui#5522] [Q&A]
<img width="427" alt="image" src="https://github.com/user-attachments/assets/d72a2cdf-2187-44ae-b084-1f6943eabb52">

on mobile in portrait, i am unable to click anything on this clerk popup. using the inspect element tool, it looks like the sidebar is "on top" of the popup. how can i make this work?

**Comment by adok0001** [comment]
Running into the same issue. @stargeneration-winston did you get a fix?

**Comment by ShamilDibirov** [comment]
I have the same problem, trying to solve it

**Comment by mk48** [comment]
This solution worked for me
https://stackoverflow.com/a/78488757

```
<UserButton
        appearance={{
          elements: { userButtonPopoverCard: { pointerEvents: "initial" } },
        }}
        afterSignOutUrl="/"
/>

```
